### ISSUE TITLE
A11y_Model Builder_Environment_Non-text Contrast: The non-text contrast ratio for the focus on "Local CPU" environment tile is less than required non-text contrast ratio 3:1.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2659]
### GitHub Tags
#WCAG1.4.11; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Non-text Contrast; #Win32;

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22621.1702

### Repro Steps

1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till data classification tile and press enter.
7. Select Training Environment screen will appear.
8. Tab till Environment tiles present on screen.
9. Inspect the focus contrast ratio with respect to selected environment tile.
10. Observe that the non-text contrast ratio for the focus on "Local CPU" environment tile is less than required non-text contrast ratio 3:1.

### Actual Result
The non-text contrast ratio for the focus on "Local CPU" environment tile is 2.231:1 i.e. less than required non-text contrast ratio 3:1.
Note:
Similar issue is observed throughout the application for selected tab/tile.

### Expected Result
Keyboard focus should be clearly visible on "Local CPU" environment tile. The color contrast ratio between the keyboard focus and the selected control background should be greater than or equal to 3:1.

### User Impact
Low vision users will not able to notify and locate the focus if it's color contrast ratio is not meeting the required ratio 3:1.

### Attachment
![Keyboard focus is not clearly visible on Local CPU environment tile](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/f5fa88f7-2c43-4f02-ad39-6ac99152296b)

Color Contrast 

![The non-text contrast ratio for the focus on Local CPU environment tile is less than required non-text contrast ratio](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/a8f7d8c1-96b0-40c0-9356-57b5db3fb682)

**Comment by zewditu** [comment]
@manaswita-chougule , ready to validate by Model Builder version 16.17.0.2332602.

**Comment by manaswita-chougule** [comment]
#Reopened; #Regressed:06-29-23;
@zewditu 
Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.3
Model Builder : Version 17.17.0.2332602

The issue is not fixed, please find below attachment for reference. Thanks!

![Screenshot 2023-06-29 125637](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/7f541f08-f4d2-430f-81d0-4b952f2736a5)

**Comment by manaswita-chougule** [comment]
#Closed; #Regressed:07-12-23;

Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.5
Model Builder : Version 17.17.0.2332602

The issue is fixed, hence closing the bug. Please find below attachment for reference. Thanks!
![The color contrast ratio between the keyboard focus and the selected control background is greater than required](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/b30da1c8-755d-4196-b7e4-59eaba7f6bf0)
